test/components/structures/ThreadView-test.tsx

```diff
@@ -108,6 +108,7 @@ describe("ThreadView", () => {
         stubClient();
         mockPlatformPeg();
         mockClient = mocked(MatrixClientPeg.get());
+        jest.spyOn(mockClient, "supportsExperimentalThreads").mockReturnValue(true);
```

Comment:
With the changes in https://github.com/matrix-org/matrix-js-sdk/pull/2856, the reason the tests failed before without this was because `room.eventShouldLiveIn(...)` would return `threadId: undefined` when `supportsExperimentalThreads` was turned off and we couldn't add the event to the timeline as expected.